stl/inc/coroutine

```diff
@@ -93,7 +93,7 @@ protected:
 };
 
 template <class _Promise>
-struct coroutine_handle : coroutine_handle<> {
+struct coroutine_handle : private coroutine_handle<> {

```

Comment:
wouldn't this break

```
void f(coroutine_handle<>);
coroutine_handle<T> h;
f(h);
```
...because (inaccessible) derived-to-base conv is prefered to user-defined conv?

Reply:
Yes - that's what I get for never compiling this.